definitions/features/katello.rb

```diff
@@ -17,7 +17,8 @@ def current_version
 
   def services
     [
-      system_service('elasticsearch', 30)
```

Comment:
Do we actually support foreman-maintain back to a version of Katello that had elasticsearch?

Reply:
I think this is safe to drop now.

Reply:
Yes, safe to drop, not expecting anyone on 6.1 version in downstream.